Automattic/simplenote-macos
# Issue/ignore config.plist

### Fix
Similar to what was [done in Android](https://github.com/Automattic/simplenote-android/pull/670), this PR moves the existing `Simplenote/config.plist` file (which contains test values) to `Simplenote/config-demo.plist`.  Additionally, the `readme.md` instructions were updated to reflect this changes. **Note:** `.gitignore` [already contains an entry](https://github.com/Automattic/simplenote-macos/blob/develop/.gitignore#L19) for `Simplenote/config.plist`.

Other changes in this PR include:
* `readme.md` → Explicitly stating Simperium is not accepting any new developer accounts at this time
* `readme.md` → Adding some extra copy to [match simplenote-ios](https://github.com/Automattic/simplenote-ios/blob/develop/README.md) 
* Adding `CONTRIBUTING.md`

### Test
1. Checkout this PR's branch
2. Clean/Build the project
3 Verify the build error below is shown:

```
Build input file cannot be found: '/{path-to-the-local-repo}/simplenote-macos/Simplenote/config.plist'
```

4. Copy `Simplenote/config-demo.plist` to `Simplenote/config.plist`.
5. Clean/Build the project
6. Notice build error ☝️ isn't shown.

### Review
@jkmassel would you be so kind to take a 👀 at this? Also @theck13 just because I updated the readme & contributing files
